src/PrestaShopBundle/Kernel/ModuleRepository.php

```diff
@@ -89,7 +91,7 @@ public function getActiveModulesPaths()
 
             foreach ($modulesFiles as $moduleFile) {
                 if (in_array($moduleFile->getFilename(), $activeModules)) {
-                    $this->activeModulesPaths[] = $moduleFile->getPathname();
+                    $this->activeModulesPaths[$moduleFile->getFilename()] = $moduleFile->getPathname();
```

Comment:
Is it expected? Why modify the deprecated class implementation?

Reply:
No reason to change it as the ModuleRepository is used now.

Reply:
Fixed